tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -15,81 +150,111 @@ do
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Running Make configure ==="
 make configure
+check_command $? "make configure"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Building firmware binary ==="
 make -C arm-none-eabi -j`nproc` firmware
+check_command $? "make -C arm-none-eabi -j`nproc` firmware"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Renaming firmware binary ==="
-mv arm-none-eabi/firmware.bin arm-none-eabi/fw_edgetx_main_tx16s_lua-ppmus-mode2_release.bin
+mv arm-none-eabi/firmware.bin arm-none-eabi/fw_${SOURCE_DIR}_${RADIO_TYPE}_lua-ppmus-mode2_release.bin
+check_command $? "mv arm-none-eabi/firmware.bin arm-none-eabi/fw_${SOURCE_DIR}_${RADIO_TYPE}_lua-ppmus-mode2_release.bin"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Building Companion ==="
 make -C native -j`nproc` companion
+check_command $? "make -C native -j`nproc` companion"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Building Simulator ==="
 make -C native -j`nproc` simulator
+check_command $? "make -C native -j`nproc` simulator"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Building radio simulator library ==="
 make -C native -j`nproc` libsimulator
+check_command $? "make -C native -j`nproc` libsimulator"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
   read
 fi
 
 echo "=== Step $((STEP++)): Making an installer ==="
 make -C native installer
+check_command $? "make -C native installer"
 if [[ $PAUSEAFTEREACHLINE == "true" ]]; then
   echo "Step finished. Please check the output above and press Enter to continue or Ctrl+C to stop."
 fi
 
-echo "Finished. You can find the TX16S firmware in ~/edgetx/edgetx_main/build-output/arm-none-eabi"
-echo "and Companion installer packet in ~/edgetx/edgetx_main/build-output/native/companion"
+echo "Finished."
+echo ""
+echo "firmware (${RADIO_TYPE}): ${SOURCE_DIR}/${BUILD_OUTPUT_DIR}/arm-none-eabi/fw_${SOURCE_DIR}_${RADIO_TYPE}_lua-ppmus-mode2_release.bin"
```

Comment:
Here similarly to lines 217 & 218 above - `arm-none-eabi/fw_${SOURCE_DIR}_$` evaluates to subfolder that does not exist.